### DISCUSSION TITLE
How to improve WIFI multicast rate?

### DISCUSSION THREAD

**Discussion by yunaang** [open-sdr/openwifi#209] [Q&A]
I know that the multicast packet is a low rate packet，if I don't care about the unicast rate, can I increase the multicast rate? 
Whether it can be achieved by modifying the driver software

thanks

**Comment by JiaoXianjun** [comment]
https://github.com/open-sdr/openwifi/blob/master/doc/app_notes/frequent_trick.md#Tx-rate-config

**Comment by yunaang** [comment]
这是来自QQ邮箱的假期自动回复邮件。 您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
